Jenkinsfile

```diff
@@ -0,0 +1,77 @@
+#!groovy
```

Comment:
I suppose we'll probably switch this to skaffold and such soon. But for now I just added a Jenkinsfile like other go services.